Update using-fonts.md (#7999)
magento/devdocs: src/guides/v2.3/frontend-dev-guide/css-topics/using-fonts.md

@icons__font-path: '@{baseDir}fonts/Theme-Icons'; //  Add you fonts in your-theme/web/fonts
@icons__font-name: 'Theme-Icons';
```

> Once you upload icons from the IcoMoon, It occurs 404 error `woff2` file icon not found in console. `woff2` Missing from this IcoMoon package. So you need to generate `woff` to `woff2` file.

Reviewer: ```suggestion
When you upload icons from the IcoMoon, a "404 error `woff2` file icon not found" may occur in the console. This means the `woff2` file is missing from the IcoMoon package. You just need to copy the `woff` file to `woff2`.
```
